ui.apps/src/main/content/jcr_root/apps/asset-share-commons/clientlibs/clientlib-dependencies/.content.xml

```diff
@@ -22,4 +22,4 @@
     allowProxy="{Boolean}true"
     categories="[asset-share-commons.dependencies]"
     embed="[granite.csrf.standalone, asset-share-commons.vendor]"
-    jsProcessor="[min:gcc,default:none]"/>
+    jsProcessor="[min:gcc;languageIn=ECMASCRIPT_2015;languageOut=ECMASCRIPT_2015,default:none]"/>
```

Comment:
I am not sure if the min settings of the embedded libraries are considered and take precedence here.